### PR TITLE
Liveness probe support

### PR DESCRIPTION
**Pull request by sliaptsou**
Issue # , if available: #333

Description of changes: Implemented liveness probe by HTTP request. Updated helm chart to allow enable/disable liveness probe.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -162,6 +172,9 @@ func ParseCliArgs() (config Config, err error) {
 	flag.StringVar(&config.UptimeFromFile, "uptime-from-file", getEnv(uptimeFromFileConfigKey, uptimeFromFileDefault), "If specified, read system uptime from the file path (useful for testing).")
 	flag.BoolVar(&config.EnablePrometheus, "enable-prometheus-server", getBoolEnv(enablePrometheusConfigKey, enablePrometheusDefault), "If true, a http server is used for exposing prometheus metrics in /metrics endpoint.")
 	flag.IntVar(&config.PrometheusPort, "prometheus-server-port", getIntEnv(prometheusPortConfigKey, prometheusPortDefault), "The port for running the prometheus http server.")
+	flag.BoolVar(&config.EnableProbes, "enable-probes-server", getBoolEnv(enableProbesConfigKey, enableProbesDefault), "If true, a http server is used for exposing probes in /probes endpoint.")
```
Comment (bwagner5) [inline pkg/config/config.go:175]: the default you set above is `/healthz` rather than `/probes` right?

**Review by bwagner5 — CHANGES_REQUESTED** [review]
nice work! Can you add the new probes configuration to the helm readme in the general config section?

File: pkg/config/config.go (hunk 1 above)
Comment (sliaptsou) [reply pkg/config/config.go:175]: Yes. I made a mistake :worried:

**Review by bwagner5 — APPROVED** [review]
LGTM, thank you!

**Pull request merged**